timeseries/src/autogluon/timeseries/evaluator.py

```diff
@@ -219,6 +240,9 @@ def save_past_metrics(self, data_past: TimeSeriesDataFrame):
             y_past=data_past[self.target_column], seasonal_period=seasonal_period
         )
 
+        if self.eval_metric == "RMSSE":
+            self._random_walk_error = in_sample_random_walk_error(y_past=data_past[self.target_column])
```

Comment:
Similar to the above discussion, something like `_past_abs_seasonal_error` and `_past_squared_seasonal_error`?

Reply:
Agreed and updated.